Utilities/types.h

```diff
@@ -84,6 +84,27 @@
 #define AUDIT(...) ((void)0)
 #endif
 
+#if defined(__cpp_lib_bit_cast) && (__cpp_lib_bit_cast >= 201806L)
+#include <bit>
+#else
+namespace std
+{
+	template <class To, class From, typename = std::enable_if_t<sizeof(To) == sizeof(From)>>
+	constexpr To bit_cast(const From& from) noexcept
```

Comment:
That's undefined behavior though. Why just not use `rpcs3::bit_cast` and forward to `std` if available?

Reply:
Knowing that it'll have to be replaced again with std::bit_cast is a mess, because there will be point of time when rpcs3::bitcast is completely obsolete. It's a compromise.